### ISSUE TITLE
SaveImage cannot find ChangeHistory

### ISSUE THREAD

**Issue by pllim** [ejeschke/ginga#330]
In the midst of working on a follow-up PR for `SaveImage` (#331), I realized that it can no longer find `ChangeHistory` global plugin. To reproduce the problem:

1. Comment out the `try ... except ...` block in `SaveImage.__init__()` so that it calls `self.history_obj = self.fv.gpmon.getPlugin('ChangeHistory')` and reports failure as error to log.
2. Install this temporary version of Ginga and run it.
3. Start `ChangeHistory` global plugin from menu.
4. Start `SaveImage` global plugin from menu.

When I inserted `print(self.fv.gpmon.getNames())` in `SaveImage.__init__()`, I only see the following listed although I see `ChangeHistory` plugin active and working:

```python
['info', 'colorbar', 'contents', 'cursor', 'zoom', 'toolbar', 'header', 'thumbs', 'operations',
'fbrowser', 'wbrowser', 'pan']
```

This does not make sense, because `SaveImage` did not have this problem before (the last time I tested it maybe 1-2 months ago).

**Comment by pllim** [comment]
@ejeschke , I see that you recently modified how things work behind the scenes. Do you see how that would affect `ChangeHistory` not showing up on `gpmon` but somehow still works?

**Comment by ejeschke** [comment]
@pllim, I do not get this error.  It should be able to find the plugin even if you haven't started it.  Is this in stock Ginga reference viewer or `stginga`?

**Comment by ejeschke** [comment]
@pllim, no major changes "behind the scenes" lately. Just a few tweaks to `freepan` handling and a few changes to pg widgets support.  Certainly nothing that should affect the loading of a plugin module.

**Comment by pllim** [comment]
@ejeschke , I think it is just the way plugins interact with each other has changed now that `SaveImage` and `ChangeHistory` are both in Ginga and not loaded by default. I'll figure out a fix. Thanks!

**Comment by pllim** [comment]
Fixed in e7f051b
